sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/MarshalSpec.kt

```diff
@@ -15,12 +15,7 @@
  */
 package com.squareup.sqldelight
 
-import com.squareup.javapoet.ClassName
-import com.squareup.javapoet.FieldSpec
-import com.squareup.javapoet.MethodSpec
-import com.squareup.javapoet.ParameterizedTypeName
-import com.squareup.javapoet.TypeSpec
-import com.squareup.javapoet.TypeVariableName
+import com.squareup.javapoet.*
```

Comment:
grab https://github.com/square/java-code-styles and reformat using that so you don't have the star import/double space tabs.